osu.Game/Scoring/ScoreInfo.cs

```diff
@@ -56,7 +55,7 @@ public BeatmapInfo BeatmapInfo
         public string BeatmapHash { get; private set; } = string.Empty;
 
         [MapTo("BeatmapInfo")]
-        private BeatmapInfo beatmapInfo { get; set; } = null!;
+        public BeatmapInfo AccessableBeatmapInfo { get; private set; } = null!;
```

Comment:
`Accessable` is not a word. Please consider using an IDE with spell check.

Let's call this `BeatmapInfoBacking`, and leave xmldoc warning against using it.

Something like:

```csharp
        /// <summary>
        /// The raw realm backing for <see cref="BeatmapInfo"/>. Do not modify directly.
        /// </summary>
        public BeatmapInfo BeatmapInfoBacking { get; private set; } = null!;
```